File: frontend-svelte/src/lib/api/models/Organization.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Project } from './Project';
import type { User } from './User';

export type Organization = {
    id?: string;
    name?: string | null;
    projects?: Array<Project> | null;
    users?: Array<User> | null;
};

File: backend/src/Transleet/Models/Organization.cs
using System.ComponentModel.DataAnnotations.Schema;


namespace Transleet.Models;


public class Organization
{
    public Guid Id { get; set; }
    public string Name { get; set; }
    public List<Project>? Projects { get; set; }
    public List<User> Users { get; set; }
}
